source/androidx.media3/media3-cast/Transforms/Metadata.xml

```diff
@@ -0,0 +1,29 @@
+﻿<metadata>
+
+    <attr
+        path="/api/package[@name='androidx.media3.cast']/class[@name='CastPlayer']/method[@name='addMediaItems' and count(parameter)=2 and parameter[1][@type='int'] and parameter[2][@type='java.util.List&lt;androidx.media3.common.MediaItem&gt;']]/parameter[2]"
+        name="managedType"
+        >
+        System.Collections.Generic.IList &lt; AndroidX.Media3.Common.MediaItem &gt;
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.cast']/class[@name='CastPlayer']/method[@name='setMediaItems' and count(parameter)=3 and parameter[1][@type='java.util.List&lt;androidx.media3.common.MediaItem&gt;'] and parameter[2][@type='int'] and parameter[3][@type='long']]/parameter[1]"
+        name="managedType"
+        >
+        System.Collections.Generic.IList &lt; AndroidX.Media3.Common.MediaItem &gt;
+    </attr>
+
+    <attr
+        path="/api/package[@name='androidx.media3.cast']/class[@name='CastPlayer']/method[@name='setMediaItems' and count(parameter)=2 and parameter[1][@type='java.util.List&lt;androidx.media3.common.MediaItem&gt;'] and parameter[2][@type='boolean']]/parameter[1]"
+        name="managedType"
+        >
+        System.Collections.Generic.IList &lt; AndroidX.Media3.Common.MediaItem &gt;
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.cast']/class[@name='CastPlayer']/method[@name='setMediaItems' and count(parameter)=2 and parameter[1][@type='java.util.List&lt;androidx.media3.common.MediaItem&gt;'] and parameter[2][@type='boolean']]"
+        name="override"
```

Comment:
There is no attribute `override`, did you mean `managedOverride`?

Reply:
yup. Typo